dvc/config.py

```diff
@@ -221,6 +221,9 @@ class Config(object):  # pylint: disable=too-many-instance-attributes
         Optional(SECTION_GCP_PROJECTNAME): str,
     }
 
+    SECTION_GDRIVE_SCOPES = "gdrive_scopes"
+    SECTION_GDRIVE_CREDENTIALPATH = SECTION_AWS_CREDENTIALPATH
```

Comment:
looks really strange, using AWS to set GDRIVE credentials. Let's rename the const name?

Reply:
It was just like `SECTION_GCP_CREDENTIALPATH = SECTION_AWS_CREDENTIALPATH` 5 lines above :). Maybe we just don't need this constants duplicated for every section? Btw, the options naming in the config schema is strange, why it is not the `OPTION_CREDENTIALPATH`? But this is out of scope of this PR.

Reply:
https://github.com/iterative/dvc/issues/2234